.github/actions/build-fixtures/action.yaml

```
@@ -23,21 +23,14 @@ runs:
       id: evm-builder
       with:
         type: ${{ steps.properties.outputs.evm-type }}
-    - name: Install solc compiler
-      shell: bash
-      run: |
-        if [ "$RUNNER_OS" == "Linux" ]; then PLATFORM="linux-amd64"; else PLATFORM="macosx-amd64"; fi
-        RELEASE_NAME=$(curl https://binaries.soliditylang.org/${PLATFORM}/list.json | jq -r --arg SOLC_VERSION "${{ steps.properties.outputs.solc }}" '.releases[$SOLC_VERSION]')
-        wget -O $GITHUB_WORKSPACE/bin/solc https://binaries.soliditylang.org/${PLATFORM}/$RELEASE_NAME
-        chmod a+x $GITHUB_WORKSPACE/bin/solc
-        echo $GITHUB_WORKSPACE/bin >> $GITHUB_PATH
     - name: Run fixtures fill
       shell: bash
       run: |
         pip install --upgrade pip
         python -m venv env
         source env/bin/activate
         pip install -e .
+        solc-select use 0.8.25 --always-install
```

I think this is required to take the solc version defined in `configs/feature.yaml`:
```suggestion
        solc-select use  ${{ steps.properties.outputs.solc }} --always-install
```